Mathlib/GroupTheory/Congruence/Hom.lean

```diff
@@ -236,22 +236,26 @@ theorem lift_apply_mk' (f : c.Quotient →* P) :
     (c.lift (f.comp c.mk') fun x y h => show f ↑x = f ↑y by rw [c.eq.2 h]) = f := by
   ext x; rcases x with ⟨⟩; rfl
 
+/-- Extensionality for maps `f, g : M⧸~ →* N`: they are equal if their composition with
```

Comment:
Is this `~` convention used elsewhere in the file? I think better to match the spelling used in the `lift_funext` docstring for now.

If you want to change the docstrings, let's do that in a second PR (and find someone else to review it)

Reply:
done